### ISSUE TITLE
Ability to research planets through the Observatory

### ISSUE THREAD

**Issue by dmodoomsirius** [zmaster587/AdvancedRocketry#966]
Can you implement the ability to research planets in the Observatory? It will allow players to start researching planets before they get into space so they have a few planets especially if a lot of the worlds are <isKnown>false</isknown>.

**Comment by Lyneira** [comment]
I agree it makes more sense to research planets in the observatory than in the warp core. You could even require the observatory to be on a space station to give people a reason to build one there.
